src/usr/local/www/interfaces.php

```diff
@@ -274,7 +274,7 @@ function remove_bad_chars($string) {
 		$pconfig['type6'] = "slaac";
 		break;
 	case "dhcp6":
-		$pconfig['dhcp6-duid'] = $wancfg['dhcp6-duid'];
+		$pconfig['dhcp6duid'] = $wancfg['dhcp6duid'];
```

Comment:
because one of the functions got upset when trying to pass 'dhcp6-duid', it thinks it's a minus